reports-api/migrations/versions/11a3360d6a01_added_audit_fields_to_all_models.py

```diff
@@ -0,0 +1,236 @@
+"""added audit fields to all models
+
+Revision ID: 11a3360d6a01
+Revises: b0b7e2536ab7
+Create Date: 2023-01-30 13:04:14.834233
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '11a3360d6a01'
+down_revision = 'b0b7e2536ab7'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column('ea_acts', sa.Column('created_by', sa.String(), nullable=True))
+    op.add_column('ea_acts', sa.Column('created_at', sa.DateTime(),server_default=sa.text('now()'), nullable=True))
+    op.add_column('ea_acts', sa.Column('updated_by', sa.String(), nullable=True))
+    op.add_column('ea_acts', sa.Column('updated_at', sa.DateTime(), nullable=True))
+    op.add_column('ea_acts', sa.Column('is_active', sa.Boolean(), server_default='t', nullable=True))
+    op.add_column('ea_acts', sa.Column('is_deleted', sa.Boolean(), server_default='f', nullable=True))
```

Comment:
The default of "is_deleted" should be false.right?

Reply:
It is false (`'f'`)